matplotcheck/raster.py

```diff
@@ -100,68 +97,62 @@ def assert_legend_accuracy_classified_image(
         Returns
         ----------
         Nothing (if checks pass) or raises error
-
-
-        Notes
-        ----------
-        First compares all_label_options against the legend labels to find
-        which element of all_label_options matches that entry. E.g. if the
-        first legend entry has a match in the first list in all_label_options,
-        then that legend entry corresponds to the first class (value 0).
-        Then the plot image array is copied and the values are set to the
-        legend label that match the values (i.e. the element in
-        all_label_options). The same is done for the expected image array.
-        Finally those two arrays of strings are compared. Passes if they match.
         """
         # Retrieve image array
```

Comment:
looking at kristen's original description... i suspect that some functionality that she had included has been removed here... so i am going to go back to her original code. specifically we want to make sure that the patches and associated labels n the plot map to the expected array correctly. ... more in a bit on this.